common/src/main/java/net/whimxiqal/journey/Journey.java

```diff
@@ -90,7 +90,7 @@ public void init() {
     BStatsUtil.register(proxy.platform().bStatsChartConsumer());
 
     if (dataManager.version() != DataVersion.latest()) {
-      logger().error("The Journey database is using an invalid version.");
+      logger().warn("The Journey database is using an old version.");
```

Comment:
At this point, the version should have been completely updated since `dataManager.initialize()` was called. So this should remain an error, and Journey should actually probably be disabled within Bukkit.